src/runtime/analytics-service.js

```diff
@@ -74,6 +79,25 @@ export class AnalyticsService {
     this.lastAction_ = null;
   }
 
+  /**
+   * @param {boolean=} override
+   */
+  setTransactionId(override = false) {
```

Comment:
We will be setting it using the actual value that comes from the payments, right? In that case, we should pass it as an argument?

Reply:
I have `setTransactionId` only in case we want to reset for some reason.
When we receive the value from the payments, we could just check if it matches existing `transactionId`

WDYT?